modules/react-components/src/components/documentation/documentation-link.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { TestableComponentInterface } from "@wso2is/core/models";
+import React, { FunctionComponent, ReactElement } from "react";
+import { Icon } from "semantic-ui-react";
+/**
+ * DocumentationLinkTag component Prop types.
+ */
+export interface DocumentationLinkTagPropsInterface extends  TestableComponentInterface { 
+    /**
+     * Documentation URL.
+     */
+    link: string;
+    /**
+     * Documentation URL target property. Opens in a new window by default.
+     */
+    target?: string;
+    /**
+     * Text on the anchor tag. Defaults to Learn More.
+     */
+    text?: string;
+}
+
+/**
+ * Documentation link anchor tag component.
+ *
+ * @param {DocumentationLinkTagPropsInterface} props - Props injected to the component.
+ *
+ * @return {React.ReactElement}
+ */
+export const DocumentationLinkTag: FunctionComponent<DocumentationLinkTagPropsInterface> = (
```

Comment:
Shall we call this `DocumentationLink`? WDYT?

Reply:
The reason to add `DocumentationLinkTag` is to avoid the confusion between the `DocumentationLinks` context object that is being used. But we can come up with an alternative name if `DocumentationLinkTag` is not suitable here.

Reply:
Fixed with a1e3fee5875cb3663bd76c58bbfe8782428355b8